.pre-commit-config.yaml

```
@@ -19,6 +19,7 @@ repos:
       name: flake8
       language_version: python3
       exclude: "__init__.py"
+      args: ["--config=setup.cfg"]
```

Feels more yaml-ic? This is what I have been using on my end.
```suggestion
      args: [--config, setup.cfg]
```